packages/manager/modules/server-sidebar/src/sidebar.constants.js

```diff
@@ -346,20 +346,19 @@ export const DEDICATED_NETWORK_CONFIG = {
     },
     {
       id: 'cloud_connect',
-      feature: 'cloud-connect',
```

Comment:
I'm not sure we should remove the `feature` rule

Reply:
If I let it, it doesn't work (I write same than vrack)

Reply:
Yes, we need to change feature rules to open it in CA. vrack doesn't use feature yet, because feature are the new way to enable/disable features based on region/sub